### PR TITLE
Add section for DDAU

### PR DESCRIPTION
**Pull request by konstankinollc**
I am totally open to your suggestions and comments as well as critique.

### CODE REVIEW

File: guides/release/components/triggering-changes-with-actions.md [hunk 1]
```diff
@@ -7,6 +16,27 @@ But what about the opposite direction? How does data flow back out of
...
+bucket with a valve at the bottom. When you refill it with water you
+are passing Data-Down into it. When you open a valve, you initiate
+Action-Up. Now your component is driven by your actions and uses the
+data it holds. In Ember, components use actions to communicate events
+and changes within your component or up to the parent component.
+
+Now you know that the child component can be used to chain activity
+from user interactions (such as a button click) and propagate that
+action for data flow within its own **actions** or out of the child
+component to the parent component. As we previously stated, recipient
```
Comment (NullVoxPopuli) [inline guides/release/components/triggering-changes-with-actions.md]: > As we previously stated

it wasn't that long ago, and I somewhat feel like I'm being accused of poor memory skills. Is there a better way to phrase this?  Maybe something like combining with the previous sentence? Maybe that'd fix the pronoun reference, too?

File: guides/release/components/triggering-changes-with-actions.md [hunk 2]
```diff
@@ -7,6 +16,27 @@ But what about the opposite direction? How does data flow back out of
...
+with triggers, data, and UI. Think about your component as an empty
+bucket with a valve at the bottom. When you refill it with water you
+are passing Data-Down into it. When you open a valve, you initiate
+Action-Up. Now your component is driven by your actions and uses the
+data it holds. In Ember, components use actions to communicate events
+and changes within your component or up to the parent component.
+
+Now you know that the child component can be used to chain activity
+from user interactions (such as a button click) and propagate that
+action for data flow within its own **actions** or out of the child
```
Comment (NullVoxPopuli) [inline guides/release/components/triggering-changes-with-actions.md]: > its

I don't know which thing its is referring to

**Comment by jenweber** [comment]
I think this is good to merge after @NullVoxPopuli's feedback is incorporated. Let us know if you have any questions. Thanks for your work!

File: guides/release/components/triggering-changes-with-actions.md [hunk 3]
```diff
@@ -7,6 +16,27 @@ But what about the opposite direction? How does data flow back out of
 the component to the parent? In Ember, components use **actions** to
 communicate events and changes.
 
+To make this easier to explain, we need to bring another design pattern
+into the scope.
+
+## Data-Down, Actions-Up
```
Comment (locks) [inline guides/release/components/triggering-changes-with-actions.md]:
```suggestion
## Data Down, Actions Up
```

File: guides/release/components/triggering-changes-with-actions.md [hunk 4]
```diff
@@ -7,6 +16,27 @@ But what about the opposite direction? How does data flow back out of
...
+
+A Data-Down, Actions-Up pattern allows you to abstract your
```
Comment (locks) [inline guides/release/components/triggering-changes-with-actions.md]:
```suggestion
A Data Down, Actions Up (DDAU) pattern allows you to abstract your
```

**Comment by locks** [comment]
@konstankinollc do you have some time to take care of the suggestions? 🙏 I tried to help it along, but I don't have permissions to commit to your branch.

File: guides/release/components/triggering-changes-with-actions.md (hunk 1 above)
Comment (jenweber) [inline guides/release/components/triggering-changes-with-actions.md]:
```suggestion
component to the parent component. The recipient
```

**Comment by jenweber** [comment]
Hi @konstankinollc, just following up to see if you are interested in making these changes so we can get this PR merged in. Thanks for your help!

**Comment by jenweber** [comment]
I'm closing this PR due to inactivity, however if you would like to resubmit this, feel free to open a new PR, and include a link to this one in the description. Thank you for your interest in helping with Ember docs!

**Pull request closed without merging**